src/app/Components/user-creation/user-creation.component.html

```diff
@@ -0,0 +1,133 @@
+<div class="container">
+  <div class="close">
+    <button mat-button matDialogClose><mat-icon>close</mat-icon></button>
+  </div>
+  <div class="dialog">
+    <div class="header" *ngIf="!create">
+      <h1 matDialogTitle>Order Type Selection</h1>
+      <h4>Please Select <strong>One (1)</strong> option below</h4>
+      <mat-dialog-actions>
+        <button mat-raised-button (click)="show()">Sign Up</button>
+        <button mat-raised-button matDialogClose routerLink="/checkout">
+          CheckOut without Account
+        </button>
+      </mat-dialog-actions>
+    </div>
+    
+
+    <!-- Create User Account -->
+    <mat-dialog-content class="dialog-content" *ngIf="create">
+      <mat-card>
+        <mat-card-title>Create Account</mat-card-title>
+        <mat-card-content>
+          <form
+            #createForm="ngForm"
+            name="createForm"
+            (ngSubmit)="getUserInformation(createForm.value)"
+          >
+            <table cellspacing="0" class="flexTable">
+              <tr>
+                <td>
+                  <mat-form-field appearance="fill">
+                    <mat-label>First name</mat-label>
+                    <input matInput type="text" name="firstName" ngModel />
+                  </mat-form-field>
+                </td>
+                <td>
+                  <mat-form-field appearance="fill">
+                    <mat-label>Last Name</mat-label>
+                    <input matInput type="text" name="lastName" ngModel />
+                  </mat-form-field>
+                </td>
+              </tr>
+            </table>
+
+            <table cellspacing="0" class="flexTable">
+              <tr>
+                <td>
+                  <mat-form-field appearance="fill">
+                    <mat-label>Address 1</mat-label>
+                    <input matInput type="text" placeholder="Ex. 100 Main St" />
+                  </mat-form-field>
+                </td>
+                <td>
+                  <mat-form-field appearance="fill">
+                    <mat-label>Parish</mat-label>
```

Comment:
parish?

Reply:
It would be great to have A more detailed way to find the address with the map.